internal/proto/buf.gen.yaml

```diff
@@ -0,0 +1,12 @@
+version: v1
+managed:
+  enabled: true
+  go_package_prefix:
+    default: "github.com/conduitio/conduit-processor-sdk/internal/proto"
+    except:
+      - buf.build/haris/conduit-commons
```

Comment:
Needs to be updated once https://github.com/ConduitIO/conduit-commons/pull/7 is merged and the proto is pushed to BSR.

Reply:
Updated.